components/view/PagesViewer.tsx

```diff
@@ -30,8 +31,14 @@ export default function PagesViewer({
   versionNumber: number;
   brand?: Brand;
 }) {
+  const router = useRouter();
   const numPages = pages.length;
-  const [pageNumber, setPageNumber] = useState<number>(1); // start on first page
+  const pageQuery = router.query.p ? Number(router.query.p) : 1;
+
+  const [pageNumber, setPageNumber] = useState<number>(() =>
+    pageQuery <= numPages ? pageQuery : 1,
```

Comment:
```suggestion
    pageQuery >= 1 && pageQuery <= numPages ? pageQuery : 1,
```

just to make sure that the user doesn't enter a negative number